packages/containerd-1.7/containerd-1.7.spec

```
@@ -8,7 +8,9 @@
 
 %global _dwz_low_mem_die_limit 0
 
-Name: %{_cross_os}%{gorepo}
+Name: %{_cross_os}%{gorepo}-1.7
+Provides: %{_cross_os}%{gorepo} = 1
```

Let's use the epoch to avoid the more complicated version comparison rules.
```suggestion
Provides: %{_cross_os}%{gorepo} = 1:
```